src/db.rs

```diff
@@ -0,0 +1,42 @@
+use postgres::{Client, Error, NoTls};
+
+pub struct Database {
+    user: String,
+    pass: String,
+    url: String,
+    port: String,
+}
+
+impl Database {
+    pub fn new(user: String, pass: String, url: String, port: String) -> Database {
+        Database {
+            user,
+            pass,
+            url,
+            port,
+        }
+    }
+    pub fn connect(&self) -> Result<(), Error> {
+        // Connect to postgresql client
+        let connection_link = format!(
+            "postgresql://{}:{}@{}:{}/pomegranate",
+            self.user, self.pass, self.url, self.port
+        );
+        println!("{:?}", connection_link);
```

Comment:
Please use log levels instead of raw `println`. Add a meaningful message to it.

(Also, add a new line before ;) )
```suggestion

        info!("Connecting to the PostgreSQL database: {:?}", connection_link);
```

Reply:
Done